### PR TITLE
Add a "what you need to do to run a conf" page.

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: docs/organizer-guide/confs/start.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    speakers,
+    budgeting,
+    and how to manage everything.
+    The most important job of the overseeing organizer is to provide a view that isn't deep in the planning and be there to say "It will work out, don't worry".
+
+There are many people that think it would be great to run a Write the Docs and I definitely understand how it might seem from the outside,
+but it is not all roses on the organization side.
+If you are going to run a Write the Docs,
+it (for better or worse) already means something to the community,
+most times it is better to run a "Write the Docs like" event with a different name.
```
Comment (plaindocs) [inline docs/organizer-guide/confs/start.rst]: Do we want some guidance on "different name"?

File: docs/organizer-guide/confs/start.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+If you `reach out to us <mailto:conf@writethedocs.org>`__,
+we will help as much as we can and you want
+-- we want more events to happen all over the
+world.
```
Comment (plaindocs) [inline docs/organizer-guide/confs/start.rst]: Do we want any guidance on spacing temporally or geographically?

File: docs/organizer-guide/confs/start.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
+I Want to Run a Write the Docs conference, Now What?
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/start.rst]: capitalization?

File: docs/organizer-guide/confs/start.rst [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  better or for worse. We run these events as not-for-profit experiments where
+  we try to isolate the things that work well and remove, anything that degrades
+  the experience. We take risks where others might be cautious, we focus on
+  shared social experiences, and we do our best to make a difference in the
+  community.
+
+  Reading about this philosophy second-hand just doesn't convey the full scope of
+  how we operate.
+
+* **You must have an established Write the Docs organizer onboard for guidance.**
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/start.rst]: "You must have an existing Write the Docs organizer mentor" (seems better to me, FWIW)

File: docs/organizer-guide/confs/start.rst [hunk 5]
```diff
@@ -0,0 +1,42 @@
+I Want to Run a Write the Docs conference, Now What?
+====================================================
+
+We have two guidelines that you should always adhere to when thinking about
+starting your own Write the Docs event:
+
+* **You must have attended a Write the Docs conference.**
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/start.rst]: conference... in the past?

File: docs/organizer-guide/confs/start.rst [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+  We do this to ensure that you know what a "Write the Docs event" means, for
+  better or for worse. We run these events as not-for-profit experiments where
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/start.rst:10]: double space

File: docs/organizer-guide/confs/start.rst [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  we try to isolate the things that work well and remove, anything that degrades
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/start.rst]: remove, > no comma needed
maybe also change remove to avoid or minimize?

File: docs/organizer-guide/confs/start.rst [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+  the experience. We take risks where others might be cautious, we focus on
+  shared social experiences, and we do our best to make a difference in the
+  community.
+
+  Reading about this philosophy second-hand just doesn't convey the full scope of
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/start.rst]: this sentence seems a bit lonesome. I would add a second sentence that says something like "Therefore, personally attending and experiencing our community is key in understanding how our community works so that you can carry the message forward when you organize an event."

File: docs/organizer-guide/confs/start.rst [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+  how we operate.
+
+* **You must have an established Write the Docs organizer onboard for guidance.**
+
+  We run Write the Docs events as a franchise model where the local team are in
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/start.rst]: as a > on a

Comment (thatdocslady) [reply docs/organizer-guide/confs/start.rst]: model,

Comment (thatdocslady) [reply docs/organizer-guide/confs/start.rst]: team are > team is or teams are

File: docs/organizer-guide/confs/start.rst [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+  control. They arrange the schedule, scope out the venue, plan the parties, and
+  organize everything else. This gives the event a local feel which is part of
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/start.rst]: organize everything else > take care of anything else?

File: docs/organizer-guide/confs/start.rst [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+  what makes it special: if you attend Write the Docs US and Write the Docs
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/start.rst]: US > USA

File: docs/organizer-guide/confs/start.rst [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+  Europe you will find significant differences -- and that is intentional.
+
+  We have this guideline so we can ensure that the conference is organized in
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/start.rst]: I would probably reword this to something more explicit similar to:

"However, to ensure some basic consistency between the different events in philisophy and structure, first-time organizers will be assigned a mentor from our council of elders <link>, who are long-time organizers and can provide guidance and answer questions that might arise during the process."

File: docs/organizer-guide/confs/start.rst [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+  a way that is compatible with the other Write the Docs events.
+
+There are many people that would love to run a Write the Docs event, but it is
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/start.rst]: I would make this into a 3rd guideline and title it something like "You must keep in mind that organizing things is hard" or something less scary but I can't think of a different phrasing now. This section is just as important (if not more) than the previous two and most of the time people trip on this and not the others...

File: docs/organizer-guide/confs/start.rst [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+important to realize that life is not all roses on the organizational side. If
+you are going to run a Write the Docs event, it is important that you understand
+what it means to the community, sometimes it is better to run a "Write the Docs
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/start.rst]: I wouldn't give people any ideas on running a similar event with a different name, not needed here

Comment (thatdocslady) [reply docs/organizer-guide/confs/start.rst]: "what it means to the community" I would expand this to mention that people invest time money and effort to prepare presentations, tutorials, pay for tickets and travel for events and there is a lot of responsibility and task juggling involved. If people are committing, they need to be prepared for it.

**Review by thatdocslady — COMMENTED** [review]
More nit-picky reviews :)

File: docs/organizer-guide/confs/start.rst (hunk 6 above)
Comment (plaindocs) [reply docs/organizer-guide/confs/start.rst:10]: which what?

**Comment by plaindocs** [comment]
I think I've adressed all of these in the latest commit. GTG as far as I'm concerned.

**Comment by ericholscher** [comment]
:+1:
